Library/Formula/harfbuzz.rb

```diff
@@ -12,8 +12,9 @@ class Harfbuzz < Formula
 
   depends_on "pkg-config" => :build
   depends_on "glib"
-  depends_on "cairo"
+  depends_on "cairo" => :optional
```

Comment:
Would prefer to keep this `:recommended` for now. Not a fan of making potentially breaking changes for end users without some revision or version bump.